base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
kinda confusing to export this when it's a rather different thing than the `pipe` function